jme3-android/src/main/java/com/jme3/app/jmeSurfaceView/JmeSurfaceView.java

```diff
@@ -946,36 +882,136 @@ public void setOnRendererStarted(OnRendererStarted onRendererStarted) {
 
     /**
      * Sets the listener that will dispatch an event when the layout is drawn by {@link JmeSurfaceView#addGlSurfaceView()}.
-     * @see JmeSurfaceView#addGlSurfaceView()
+     *
      * @param onLayoutDrawn the event to be dispatched.
+     * @see JmeSurfaceView#addGlSurfaceView()
      */
     public void setOnLayoutDrawn(OnLayoutDrawn onLayoutDrawn) {
         this.onLayoutDrawn = onLayoutDrawn;
     }
 
     /**
      * Gets the current device GL_ES version.
+     *
      * @return the current gl_es version in a string format.
      */
     public String getGlEsVersion() {
         return configurationInfo.getGlEsVersion();
     }
 
+    /**
+     * Checks whether the error dialog is enabled upon encountering exceptions/errors.
+     * Default is : true.
+     *
+     * @return true if the error dialog is activated, false otherwise.
+     */
+    public boolean isShowErrorDialog() {
+        return showErrorDialog;
+    }
+
     /**
      * Determines whether the error dialog would be shown on encountering exceptions.
      * Default is : true.
+     *
      * @param showErrorDialog true to activate the error dialog, false otherwise.
      */
     public void setShowErrorDialog(boolean showErrorDialog) {
         this.showErrorDialog = showErrorDialog;
     }
 
     /**
-     * Checks whether the error dialog is enabled upon encountering exceptions/errors.
-     * Default is : true.
-     * @return true if the error dialog is activated, false otherwise.
+     * Determines whether the app context would be destructed
+     * with the holder activity context in case of {@link DestructionPolicy#DESTROY_WHEN_FINISH} or be
+     * spared for a second use in case of {@link DestructionPolicy#KEEP_WHEN_FINISH}.
+     * Default value is : {@link DestructionPolicy#DESTROY_WHEN_FINISH}.
+     *
+     * @see JmeSurfaceView#setDestructionPolicy(DestructionPolicy)
      */
-    public boolean isShowErrorDialog() {
-        return showErrorDialog;
+    public enum DestructionPolicy {
```

Comment:
Due to code re-formating, those are moved to the bottom of the class, so the class constructor are always at top as java conventions states : https://www.oracle.com/java/technologies/javase/codeconventions-fileorganization.html#1852